contracts/StablecoinConverter.sol

```diff
@@ -231,6 +234,12 @@ contract StablecoinConverter is EpochTokenLocker {
         }
     }
 
+    function evaluateUtility(uint128 execBuy, Order memory order) internal view returns(uint128) {
+        // Utility = ((execBuyAmt * order.sellAmt - preFeeSell * order.buyAmt) * price.buyToken) / order.sellAmt
+        uint128 preFeeSell = uint128(execBuy.mul(currentPrices[order.buyToken]).div(currentPrices[order.sellToken]));
```

Comment:
I would use uint256 here, as in the next line everything will be computed with uin256 anyways, if we use safemath